src/components/templates/loginForm/loginForm.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import InputText from '../../atoms/inputs'
+import ButtonComponent from '../../atoms/button'
+import {Form} from 'semantic-ui-react'
+import { action } from '@storybook/addon-actions';
+import { StyledFormField } from './style/loginForm.styled.component';
+
+
+export type loginFormInterface = {
+    isDisabled:boolean,
+    isPrimary:boolean,
+    isLoading:boolean,
+    isError?:boolean,
+}
+
+const onHandleSubmit = (event:any) => {
+    console.log(event,event.target.elements.email.value);
+    action('onSubmit')
+}
+
+const LogInForm = (
+    {
+        isDisabled = false,
+        isPrimary = true,
+        isLoading = false,
+        isError = false,
+    }:loginFormInterface) => {
+    return(
+        <>
+         <Form onSubmit={onHandleSubmit}>
+            <Form.Group widths="equal">
+                <StyledFormField
+                control={InputText}
+                fluid
+                label="Email"
+                pattern="[A-Za-z0-9._%+-]{2,}@[a-zA-Z]{1,}([.]{1}[a-zA-Z]{2,}|[.]{1}[a-zA-Z]{2,}[.]{1}[a-zA-Z]{2,})"
+                placeholder="Email"
+                type="email"
+                isDisabled={isDisabled}
+                disabled={isDisabled}
+                isError={isError}
+                error ={isError}
+                isErrorProp={isError}
+                name="email"
```

Comment:
Is it possible to perform validation upon every keypress instead of every form submission instead? I think its more practical that way so the user will not have to correct a lot of fields every time they submit the form

Reply:
> Is it possible to perform validation upon every keypress instead of every form submission instead? I think its more practical that way so the user will not have to correct a lot of fields every time they submit the form

Yes i agree on this, I was thinking to refactor this part since it is using a HTML5 validation, since we are using Formik I think we will use that library rather the in the box validation.